### ISSUE TITLE
Убрать засерение заминусованных комментариев.

### ISSUE THREAD

**Issue by nin-jin** [limonte/dear-habr#18]
Ибо приходится без конца тыкать в них, чтобы прочитать. Какой смысл в этом засерении? Если кто-то не хочет читать заминусованные комментарии (покажите мне хоть одного такого человека), то он может просто их проматывать - рейтинг коммента и так хорошо виден.

**Comment by limonte** [comment]
Этот функционал можно сделать опциональным - галочка в настройках аккаунта.

**Comment by nin-jin** [comment]
Зачем делать опциональным бесполезный функционал?

**Comment by limonte** [comment]
Это вам бесполезный, а мне полезный :)

**Comment by nin-jin** [comment]
Если бы они скрывались - может и был бы кому-то "полезен". А так - вам в любом случае эти коментарии проматывать. А если далее в этой ветке незаминусованные комментарии, то суть их будет не ясна без чтения заминусованных.

**Comment by acuna-personal** [comment]
Насчет опциональности - имеется ввиду опция в настройках делать ли эти комменты серыми, ибо эта опция может быть и полезна, но не всем, поэтому консенсус был бы оставить ее на усмотрение каждого. Не хочется тыкать мышкой в коммент, чтобы прочитать - отключи это засерение в настройках. Хочешь, чтобы заминусованные комменты были серыми - включи. Лично мне тоже не удобно глаза сажать, когда не видно ничего. И так видно, что он заминусован, но почему-то не удобно должно быть обычным юзерам(

**Comment by akastargazer** [comment]
> Какой смысл в этом засерении?
Засерение как бы отодвигает текст от общего обсуждения. Смысл очень простой - показать человеку, что его мнение неинтересно. Сказать более прямо - это аналог затыкания рта, в дополнение к минусованию кармы.

Принципы социального доминирования и больше ничего. Нерды думают, что это "всё как в жизни", но в реальной жизни тебе не приглушают голос и не делают прозрачным лицо. И в реальной жизни ты видишь, кто тебя минусует.

**Comment by nin-jin** [comment]
Хочется что-то "показать" конкретному человеку - ему и засеривайте. Остальные-то чем провинились?

К тому же, рейтинг комментария отражает не "интересно/не интересно", а "согласен/не согласен". Большинство заминусованный комментариев очень даже интересные, даже если вы в итоге с ним не согласитесь.

**Comment by akastargazer** [comment]
Ну так и засеривают. Скажешь что-то против Илона Маска - сначала заминусуют твои комментарии, а потом и карму. Потому что Маск - божество, его нельзя критиковать. А если ты критикуешь, тебя затопчут.
Нельзя требовать от людей, чтобы они были хорошими. В толпе работают законы толпы, а не правила общения в кофейне, например.

**Comment by Jaf4** [comment]
Выбираю и читаю только комментарии, отличающиеся количеством плюсов.

**Comment by nin-jin** [comment]
@akastargazer речь не о толпе, а о владельцах площадки. Если бы хотелось показать пользователю, что толпе не нравится его читать, то можно было бы показывать это конкретно ему. Для пущего эффекта, можно ещё выводить это дело салатовыми буквами на алом фоне.

**Comment by nin-jin** [comment]
@Jaf4 а вам нужна сортировка по числу плюсов.

**Comment by GreLI** [comment]
Иногда засерение адекватного комментария со случайным `−1` мерзко выглядит. Я бы предложил убрать засерение при `−1` и `−2`, которые часто носят случайный характер.

Для сильно заминусованных и длинных комментариев имеет смысл добавить ограничение высоты, но с возможностью раскрытия и зачернения, чтобы комментарий показывался нормально (и обратного закрытия на случай «дайте мне это развидеть!»)

**Comment by nin-jin** [comment]
Да-да, давайте ещё усложним чтение веток комментариев, где каждое второе сообщение заминусовано.

**Comment by redmanmale** [comment]
Было бы удобно вообще скрывать/схлопывать заминусованные комментарии, а то сейчас если в первой ветке развели срач, то до следующего комментария первого уровня иногда приходится листать по несколько экранов.
Или вынести это в настройки профиля.

**Comment by nin-jin** [comment]
Вы хотели сказать "схлопывать не интересные ветки"? Хорошая идея!

**Comment by skssxf** [comment]
На мой взгляд, не стоит сразу скрывать цветом комментарий, получивший минус или два. Мнения людей могут быть разными, минус может быть эмоциональным. Но серый комментарий, он как красная тряпка: у читателей включается набор психологических установок резко увеличивающих вероятность получить ещё минусов.
